core/src/edu/asu/c3simulator/screens/MainHub.java

```diff
@@ -0,0 +1,223 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.viewport.StretchViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+//import edu.asu.c3simulator.widgets.CornerAdvisor;
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+//import edu.asu.c3simulator.widgets.PlayerStatusDisplay;
+
+/**
+ * This class displays the main hub, allowing the player to navigate to their
+ * Businesses. Located after: DifficultySelection, and or HomeButton.
+ * 
+ * @author Reigel, Justin
+ */
+public class MainHub implements Screen
+{
+
+	private static final String ADVISOR_TEXT = "This is a test of TextAreaX. This is intended to cover multiple lines at a width of 200px. This is the second extention";
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	private static final int DESIGN_SCREEN_CENTER_X = DESIGN_WIDTH / 2;
+	private static final int DESIGN_SCREEN_CENTER_Y = DESIGN_HEIGHT / 2;
+
+	@SuppressWarnings("unused")
+	private Game game;
```

Comment:
Why is this still unused? Transition functionality seems very important for the main **hub**